t/10-validation.t

```diff
@@ -70,14 +70,58 @@ subtest 'create object' => sub {
   ok ($v->use_cram, 'cram files are used by default');
   is ($v->file_extension, 'cram', 'default file extension is cram');
   is ($v->index_file_extension, 'crai', 'default index file extension is crai');
-  is ($v->min_keep_days, 14, '14 days after qc complete data to ve retained');
```

Comment:
Should this go back in to test the default? Or be removed completely?

Reply:
This test cannot be run in the context of 'create object' sub-test since the build method uses a list of products, which cannot  be computed with limited input this object gets. Changing this test to show that the value can be overwritten.